prov/rxd/src/rxd_cq.c

```diff
@@ -376,7 +376,7 @@ static int rxd_match_msg(struct dlist_entry *item, const void *arg)
 	struct rxd_match_attr *attr = (struct rxd_match_attr *) arg;
 	struct rxd_x_entry *rx_entry;
 
-	rx_entry = container_of(item, struct rxd_x_entry, entry);
+	rx_entry = rxd_set_x_entry(item, rx_entry);
```

Comment:
This is not a useful change.  It's obscuring the code and the name "set_x_entry" is actually misleading.  Nothing is being set.  We're simply taking the container_of, which was obvious from the previous code.

Reply:
@shefty The purpose of the `rxd_set_x_entry` was making common getting the `container_of` of `struct rxd_x_entry` and supporting the code style.(I was following an idea of `rxd_ep_rx_cq`, `rxd_ep_tx_cq` and etc). I just explain you my thoughts.  If you consider that it is obscuring the code, I will understand you.
By the way, do you also disagree with this [01fc5df](https://github.com/ofiwg/libfabric/pull/5181/commits/01fc5df3b8a4f5cce7086f1db1c7a02e5a2cb5aa)? Here is I implement `rxd_pool_create_attrs` in order to summarize the common part of the code for creating attributes for `rxd_pkt\entry_pool_create` functions. 
Thanks.